Snakefile

```diff
@@ -36,6 +36,26 @@ rule compile_cost_assumptions:
         "scripts/compile_cost_assumptions.py"
 
 
+rule compile_cost_assumptions_usa:
+    input:
+        cost_files_to_modify=expand("outputs/costs_{year}.csv", year=config["years"]),
+        nrel_atb_input_files=expand(
+            "inputs/atb_e_{year}.parquet",
+            year=config["nrel_atb"]["nrel_atb_input_years"],
+        ),
+        nrel_atb_input_discount_rate="inputs/discount_rates_usa.csv",
+        nrel_atb_input_fuel_costs="inputs/fuel_costs_usa.csv",
+    output:
+        expand("outputs/US/costs_{year}.csv", year=config["years"]),
+    threads: 1
+    resources:
+        mem=500,
+    conda:
+        "environment.yaml"
```

Comment:
I don't know why, but `snakemake` failed over this line (my `conda` version is outdated), despite NOT setting the `--use-conda`  for `snakemake` (bug?). But we can keep it.